src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -38,7 +38,8 @@ export const ConversationSpace: React.FunctionComponent = () => {
 
   const tables = venue?.config?.tables ?? TABLES;
 
-  const venueUsers = users.filter(
+  // @debt Refactor this
+  const venueUsers = recentWorldUsers.filter(
     (user) =>
       user.lastSeenIn?.[venue.name] > (nowMs - LOC_UPDATE_FREQ_MS * 2) / 1000
```

Comment:
I think this logic can be removed because it is already in the hook you created for `useRecentWorldUsers`

Reply:
I don't think it is similar to recentWorldUsers. I don't use `lastSeenIn` in recentWorld users so far. It seems that it is a bit broken now(it rewrites the whole object, when changed, rather than updating it). Will solve this issue in `counting PR`

Reply:
If you implement `recentVenueUsers(venueId)` as I suggest in [TODO:linkToCommentWhenReviewIsPosted] (which I feel is relevant to this PR) then we can fix this up here in this PR.

Reply:
This would be resolved by a small hook such as `recentUsersInRoom(venueId, roomName)`, which would probably consume `recentUsersInVenue(venueId)`

Reply:
At the firestore level a room is just a `venue`, but I have seen code that seems to use `lastSeenIn` like follows, which makes me think that `roomTitle` may be relevant as well:

```typescript
users?.filter((user) => user.lastSeenIn?.[`${venueName}/${roomTitle}`])
```